website/src/pages/jobs.tsx

```diff
@@ -310,12 +298,30 @@ const CareersPage = props => {
                                     You strive to continuously grow and learn by genuinely soliciting feedback early and
                                     often, and humbly reflecting on your past mistakes.
                                 </p>
-                                <img src="../careers/7growth.svg" />
+                                <img src="/careers/7growth.svg" />
                             </div>
                         </div>
                     </Carousel.Item>
                 </Carousel>
 
+                <Carousel
+                    className="pb-lg-7 bg-white"
+                    prevIcon={<ArrowLeftIcon color="#808080" />}
+                    nextIcon={<ArrowRightIcon color="#808080" />}
+                >
+                    {new Array(6).fill(null).map((el, i) => (
+                        <Carousel.Item
+                            key={`review-${i}`}
+                            className="pt-sm-6 pt-5 height-800 height-md-650 height-sm-575"
```

Comment:
I think we should vertically center the child img elements since they're all different heights

Reply:
The Bootstrap React Carousel component is being stubborn with this, because there's a nested wrapper (`.carousel-inner`) above the `Carousel.Item` and it's positioned `relative`. So to get the vertical alignment I need to bring the [custom class back in](https://github.com/sourcegraph/about/pull/5213/commits/8c1f43535d8e6f99db5c7d61464f87e42803ef85), not using the Bootstrap classes because they can't reach it.

But also with this, I'm seeing the images bounce around sometimes, are you? >:( Any ideas what to do from here?

![](https://media.giphy.com/media/LJf46UpLD9ghaQHnGX/giphy.gif)

Reply:
@st0nebraker @bretthayes Is it possible to make them all the same size to avoid that overlap over the carousel nav at the bottom?
![Screenshot 2022-03-31 at 15 51 44](https://user-images.githubusercontent.com/88658267/161071703-dd8b2a49-89d8-44f6-8ce3-1d7f7d71c9e4.png)
?

Reply:
Omg, the extra <div> wrapper for centering FTW 🤦‍♀️  Thank you Brett 🙌